sphinxcontrib/drawio/__init__.py

```diff
@@ -293,6 +295,8 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_config_value(
         "drawio_builder_export_format", DEFAULT_BUILDER_EXPORT_FORMAT, "html", dict
     )
+    app.add_config_value("drawio_builder_export_directory", "srcdir", "html",
+                         ENUM("srcdir", "outdir"))
```

Comment:
I think _drawio_export_directory_ would be a better name. _builder_ in _drawio_builder_export_format_ is meant to indicate that you can specify the export format for each builder (that's why it accepts a dict instead of a string). The export directory is independent of the builder.

Reply:
makes totally sense to me - renamed the option